From 92a663dd55917fdfad5ae4daa173a9d18df0ee9e Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Wed, 18 Aug 2021 19:14:28 -0700
Subject: [PATCH 2/5] IGC/VectorCompiler/CMakeLists.txt: link to external
 LLVMGenXIntrinsics

By default LLVMGenXIntrinsics is to be build In-tree, but we want to
link externally.

Upstream-Status: Inappropriate [configuration specific]

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 IGC/VectorCompiler/CMakeLists.txt | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/IGC/VectorCompiler/CMakeLists.txt b/IGC/VectorCompiler/CMakeLists.txt
index 0b26072f0..bf1cd939d 100644
--- a/IGC/VectorCompiler/CMakeLists.txt
+++ b/IGC/VectorCompiler/CMakeLists.txt
@@ -43,22 +43,6 @@ if(LLVM_ON_WIN32)
 endif()
 
 # --- VC Intrinsics ---
-
-if(DEFINED VC_INTRINSICS_SRC)
-  set(INTRSRC "${VC_INTRINSICS_SRC}/GenXIntrinsics")
-endif()
-
-if(NOT DEFINED INTRSRC)
-  set(INTRSRC "${CMAKE_CURRENT_SOURCE_DIR}/../../../vc-intrinsics/GenXIntrinsics")
-endif()
-
-message(STATUS "[VC] Using vc-intrinsics source from: ${INTRSRC}")
-# Trick intrinsics.
-set(BUILD_EXTERNAL YES)
-# We are using prebuilt SPIRV and building intrinsics.
-set(INTRBUILD "${CMAKE_CURRENT_BINARY_DIR}/intrbuild")
-add_subdirectory(${INTRSRC} ${INTRBUILD} EXCLUDE_FROM_ALL)
-
 include(cmake/spirv.cmake)
 
 set(IGC_OPTION__VC_DISABLE_BIF_DEFAULT OFF)
-- 
2.20.1

